neuvueclient/__init__.py

```diff
@@ -888,13 +888,14 @@ def post_task_broadcast(
             raise RuntimeError("Failed to post task") from e
         return res.json()
 
-    def patch_task(self, task_id: str, overwrite_opened: bool = True, **kwargs):
+    def patch_task(self, task_id: str, author:str, overwrite_opened: bool = True, **kwargs):
```

Comment:
I think having author be a required arg will be breaking change for a lot of notebooks and app code. Could we instead default this arg to the existing author in the task, or place a conditional of when assignee or status is patched s.t. if author is not specified it either raises an exception or produces a warning that its using the existing author of the task.

Reply:
Fixed